core-dev/install_drush.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+#ddev-generated
+# This script installs drush/drush without changing the git
+# status of the Drupal checkout
+
+set -eu -o pipefail
+
+if command -v drush >/dev/null; then
+  echo "drush is already installed, taking no action. You can remove it if you want to reinstall"
+  exit
+fi
+
+echo "Installing drush without affecting git status"
+
+# Make certain that we have something staged so we can create stash
+touch .makedrush.txt
+git add .makedrush.txt
+
+# Save the stash, which will include anything people were doing
+# plus the .makedrush.txt. This gets us back to "no changes"
+# in `git status`
+git stash
+
+# Install drush
+composer require drush/drush
```

Comment:
Do we need `--with-dependencies` here?

```suggestion
composer require drush/drush --with-dependencies
```